data_steward/cdr_cleaner/cleaning_rules/create_person_ext_table.py

```diff
@@ -56,7 +70,7 @@ def __init__(self, project_id, dataset_id, sandbox_dataset_id):
         DO NOT REMOVE ORIGINAL JIRA ISSUE NUMBERS!
```

Comment:
@amcgrenera-vumc One thing, add the `table_namer` parameter to the initialization signature on line 64 and pass it to the super class in line 73 call.  So we can start using shared sandboxes.

Reply:
Let me know if I've added correctly.